src/main/webapp/WEB-INF/tags/instructor/results/remindButton.tag

```diff
@@ -0,0 +1,13 @@
+<%@ tag description="instructorFeedbackResults - remind button" %>
+<%@ tag import="teammates.common.util.Const" %>
+
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ attribute name="remindButton" type="teammates.ui.template.InstructorFeedbackResultsRemindButton" required="true" %>
+
+<div style="display: inline-block; padding-right: 5px;" class="remind-no-response">
+    <a href="#" data-actionlink="${remindButton.urlLink}" class="${remindButton.className}" data-toggle="modal"
+            <c:if test="${remindButton.disabled}">disabled="disabled"</c:if> data-target="#remindModal">
```

Comment:
Just using `disabled=""` should be fine, but best to hold off on this for now while waiting for clarification.

@damithc looking at our HTML style guide, it seems that the method `disabled="disabled"` is preferred, however most of the code base uses `disabled` (less common, <10%) or `disabled=""` (more common, >= 90%), while `disabled="disabled"` only appears once (excluding this line). Which convention should we follow?